app/2.0/docs/tools/glossary.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Polymer Tools Glossary"
+---
+
+<!-- toc -->
+
+# A
+## app shell
+In a progressive web app, the app shell contains the minimum CSS, HTML and JavaScript to load the basic UI. [Polymer CLI](#polymer-cli) is designed to work with apps that use an app shell architecture. See [Instant Loading Web Apps with an Application Shell Architecture](https://developers.google.com/web/updates/2015/11/app-shell) for more information.
+
+# B
+## bower
+bower is a package manager for the Web. The Polymer toolchain uses bower to manage and install elements for reuse in apps and in other elements. See [bower.io](https://bower.io) for more information.
+
+## bower.json
+`bower.json` is a manifest used by bower for configuring packages for use as dependencies. See [the `bower.json` specification](https://github.com/bower/spec/blob/master/json.md) for more information.
+
+# C
+## crisper
+crisper extracts inline scripts from an HTML file. This is useful if you need to run a transformation on your JavaScript, such as a minifier or a transpiler. You could also use crisper to separate JavaScript from HTML in order to deploy in an environment that uses CSP (Content Security Policy).
+
+# D
+## dependency
+In the context of Polymer apps, a dependency is a resource (a file or set of files) required by your application, such as HTML files or JavaScript libraries. Dependencies should be declared in [`polymer.json`](#polymer-json) to ensure they are included in your build. See also: [dynamic dependency](#dynamic-dependency), [static dependency](#static-dependency), [transitive dependency](#transitive-dependency).
+
+In the context of the Polymer data system, a dependency is a piece of data that an observer, computed property or computed binding depends on.
+
+## dependency variant
+Polymer dependencies vary based on the version of Polymer in which you are implementing your application or element. In the context of Polymer tools, a variant is a version of your app that is built using a specific version of Polymer. 
+
+If you run `polymer install --variants`, the `polymer install` tool creates separate builds for your app or element using the versions of Polymer you specify. You can use the [Polymer development server](#polyserve) to test these variants. 
+
+## dynamic dependency
+A dynamic dependency is a dependency that is imported at runtime, using a mechanism such as `importHref`. Compare [static dependency](#static-dependency).
+
+# E
+## eager loading
+Eager loading refers to the process of loading dependencies statically, rather than on demand. Compare [lazy loading](#lazy-loading).
+
+## entry point
+The HTML actually loaded by the browser to start the application. Usually `index.html`. In an application built with the [app shell](#app-shell) architecture, the entrypoint is loaded for many different URLs.
+
+# F
+## fragment
+A lazily-loaded HTML file. Fragments are used to define bundles and are often associated with routes or non-critical resources.
```

Comment:
- A Fragment is much more about defining a bundle than it is about a lazy-loaded HTML file. I would reorder these two sentances so that we don't bury the lead here.